webapp.js

```diff
@@ -49,12 +48,7 @@ app.use(express.static('dist'));
 app.use(bodyParser.json());
 
 app.get('/api/data', function(req, res) {
```

Comment:
If you wanted to be a little clever, I think you could have `dataManager.loadData` take `db` as the first argument, then pass a bound callback directly, instead of wrapping it in an extra anonymous function here.

So, the signature for `loadData` would be:
```
function loadData(db, req, res) {
  ...
}
```

Then, here would just be:
```
app.get('/api/data', dataManager.loadData.bind(null, db);
```

You could do the same for all the other endpoints that callback to a function from the `dataManager` module.